CMakeLists.txt

```diff
@@ -231,6 +240,10 @@ cprover_default_properties(
     xml)
 
 option(WITH_JBMC "Build the JBMC Java front-end" ON)
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL Darwin)
+    # See https://github.com/diffblue/cbmc/issues/6343
+    message(STATUS "Use -DWITH_JBMC=OFF if jbmc fails to build on MacOS")
```

Comment:
I think (until we can fix the models library) this would be better as a check against the Java version, and then print a warning message and automatically set WITH_JBMC=OFF. Ultimately this is not about Darwin or any particular OS/architecture - its purely a Java version/API change.

Reply:
@chrisr-diffblue Done.